lib/plugin.dart

```diff
@@ -331,7 +319,7 @@ class AtomDartPackage extends AtomPackage {
       // show infos and todos
       'configureErrorsView': {
         'title': "Errors view configuration",
-        'description': 'Choose which types of items to show in the errors view.',
+        'description': 'Choose which types of items to show in Linter.',
```

Comment:
Honestly, this doesn't read quite right to me; `in the linter view.` I think would read better, eps. in the context of the other settings. However, we can land this and iterate.